meson.build

```diff
@@ -0,0 +1,47 @@
+project('gc', 'c', version: '7.6.8',
+    meson_version: '>=0.47.0')
```

Comment:
Please specify project license here, e.g. https://github.com/mesonbuild/catch2/blob/2.4.1/meson.build#L4

Reply:
I'm not sure if it's a standard license. It looks like a BSDish one https://github.com/ivmai/bdwgc/blob/master/README.QUICK

Reply:
I think specifying `custom` is fine in that case.